woocommerce/woocommerce
# Only trim multiselect buttons

If we trim everything we lose indentation in the dropdown lists. We only need to trim the buttons created when elements are selected in a multiselect.

To test: 
Go to Shipping Zones country dropdown.
Observe indentation is present in dropdown.
Observe indentation is not present in the button when an option is selected.
